### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RomanHrytsiuk**
https://RomanHrytsiuk.github.io/layout_moyo-header/
https://RomanHrytsiuk.github.io/layout_moyo-header/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,52 @@
-/* Styles go here */
+body {
+  font-family: Roboto, Helvetica, sans-serif;
```
Comment (AlisaVlasova) [inline src/style.css:2]: move it to html selector

File: src/style.css [hunk 2]
```diff
@@ -1 +1,52 @@
...
+
+.image {
+  display: flex;
+  padding: 10px 0 10px 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 10px 0;
```
Comment (AlisaVlasova) [inline src/style.css:22]: this is redundant

File: src/style.css [hunk 3]
```diff
@@ -1 +1,52 @@
...
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+  margin: 0;
```
Comment (AlisaVlasova) [inline src/style.css:40]: reset margin on first link of the list, not the active one

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 2 above)
Comment (AlisaVlasova) [reply src/style.css:20]: you can remove padding
 and set margin to 0

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 4]
```diff
@@ -1 +1,59 @@
...
+  color: #00acdc;
+}
+
+.nav__item .nav__link:first-of-type {
+  margin: 0;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```
Comment (ivanchuk-yurii) [inline src/style.css:47]: it's better to move colors into variables

File: src/style.css [hunk 5]
```diff
@@ -1 +1,59 @@
...
+  text-transform: uppercase;
+  padding: 23px 0;
+  margin-left: 20px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__item .nav__link:first-of-type {
```
Comment (ivanchuk-yurii) [inline src/style.css:41]: don't use nested selectors if not necessary

File: src/style.css [hunk 6]
```diff
@@ -1 +1,59 @@
...
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 23px 0;
+  margin: 0;
+}
+
+.image {
+  display: flex;
+  padding: 10px 0 10px 50px;
```
Comment (ivanchuk-yurii) [inline src/style.css:26]: vertical padding is useless, add left padding to `header` instead of here

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 4 above)
Comment (RomanHrytsiuk) [reply src/style.css:50]: I don't know yet, how it works.

File: src/style.css [hunk 7]
```diff
@@ -12,6 +12,7 @@ body {
   justify-content: space-between;
   align-items: center;
   padding-right: 50px;
+  padding-left: 50px;
```
Comment (ivanchuk-yurii) [inline src/style.css:15]:
```suggestion
  padding: 0 50px;
```

File: src/style.css [hunk 8]
```diff
@@ -23,25 +24,27 @@ body {
...
-  padding: 10px 0 10px 50px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
 }
```
Comment (ivanchuk-yurii) [inline src/style.css:35]:
```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```

**Review by ivanchuk-yurii — APPROVED** [review]

**Pull request closed without merging**